src/components/Calculator.js

```diff
@@ -17,40 +17,46 @@ const Calculator = () => {
   };
 
   return (
+    <main className="PageContainer">
+      <div className="content">
+        <p>Let&apos;s do some math!</p>
+      </div>
 
-    <section className="calculator-container">
-      <div className="display">{ properties.next || properties.operation || properties.total || 0}</div>
+      <section className="calculator-container">
+        <div className="display">{ properties.next || properties.operation || properties.total || 0}</div>
+
+        <div className="list-btn">
+          <button onClick={handleClick} className="list-btn-item" type="button">AC</button>
```

Comment:
### OPTIONAL
- [ ] You are using many buttons. You can create a button component for use anywhere. It will be more clear.